diff --git a/build/index.js b/build/index.js
index bbdc07b3c1f1b8e447cd8b593c6e60fbcbd8db1f..a5790da9909df67011c9c6a538a6246383fcc5bc 100644
--- a/build/index.js
+++ b/build/index.js
@@ -34,6 +34,9 @@ class Stdout extends EventEmitter {
     get columns() {
         return 100;
     }
+
+    ref() {}
+    unref() {}
 }
 class Stderr extends EventEmitter {
     constructor() {
@@ -66,6 +69,9 @@ class Stderr extends EventEmitter {
             value: () => this._lastFrame
         });
     }
+
+    ref() {}
+    unref() {}
 }
 class Stdin extends EventEmitter {
     constructor() {
@@ -98,6 +104,9 @@ class Stdin extends EventEmitter {
     pause() {
         // Do nothing
     }
+
+    ref() {}
+    unref() {}
 }
 const instances = [];
 export const render = (tree) => {
